internal/pkg/utils/testhelper/terminal/terminal.go

```diff
@@ -17,9 +17,9 @@ type Console interface {
 	// terminal device.
 	Tty() Tty
 	// Send writes string s to Console's tty.
-	Send(s string) (int, error)
+	Send(s string) error
```

Comment:
Nikde sme dane hodnoty nepouzivali, asserty boli zbytocne na 2 riadny, bolo to menej prehladne.